daprdocs/content/en/operations/hosting/kubernetes/kubernetes-persisting-scheduler.md

```diff
@@ -6,12 +6,15 @@ weight: 50000
 description: "Configure Scheduler to persist its database to make it resilient to restarts"
 ---
 
-The [Scheduler]({{< ref scheduler.md >}}) service is responsible for writing jobs to its embedded database and scheduling them for execution.
-By default, the Scheduler service database writes this data to an in-memory ephemeral tempfs volume, meaning that **this data is not persisted across restarts**. Job data will be lost during these events.
+The [Scheduler]({{< ref scheduler.md >}}) service is responsible for writing jobs to its embedded Etcd database and scheduling them for execution.
+By default, the Scheduler service database writes this data to a Persistent Volume Claim of 1Gi of size using the cluster's default [storage class](https://kubernetes.io/docs/concepts/storage/storage-classes/). It means that there is no additional parameter required to run the scheduler service reliably on most Kubernetes deployments, although you will need additional configuration in some deployments or for a production environment.
```

Comment:
```suggestion
By default, the Scheduler service database writes this data to a Persistent Volume Claim of 1Gb of size using the cluster's default [storage class](https://kubernetes.io/docs/concepts/storage/storage-classes/). This means that there is no additional parameter required to run the scheduler service reliably on most Kubernetes deployments, although you will need additional configuration in some deployments or for a production environment.
```

Reply:
@elena-kolevska is taking these changes to her PR.

Also, it is 1Gi or (1GiB), not 1Gb.

Reply:
Gi is the same a Gb. And is this PR now redundant due to this PR https://github.com/dapr/docs/pull/4306 anyway?